cmd/immuclient/command/commandline.go

```diff
@@ -28,6 +29,10 @@ func (cl *commandline) ConfigChain(post func(cmd *cobra.Command, args []string)
 		}
 		cl.options = immuc.Options().WithTokenFileName("token")
 		cl.immucl, err = immuc.Init(cl.options)
+		if err != nil {
+			return err
+		}
+		cl.immucl.WithRevisionSeparator(viper.GetString("revision-separator"))
```

Comment:
not part of options?

Reply:
It's messed up - options used by immuclient binary are used in options for the immudb client object. I'll try to extract those immuclient-specific options to a separate structure.

Reply:
Extracted separate immuclient options in 626658965929b90deccda40e5998ac6394569562,

I also added a follow-up PR to remove unnecessary fields from pkg/client options in https://github.com/codenotary/immudb/pull/1206 - but that's something to discuss since it introduces a bit of backwards incompatibility